justfile

```diff
@@ -219,6 +223,26 @@ _dry-run-release: clean deps dart-build
 dry-run-release:
      @CI=true {{just_executable()}} _dry-run-release
 
+compile-android target:
+    # See also: https://developer.android.com/studio/projects/gradle-external-native-builds#jniLibs
+    cd "$RUST_WORKSPACE"; \
+        cargo ndk -t $(echo "{{target}}" | sed 's/[^ ]* */&/g') -p $ANDROID_PLATFORM_VERSION -o "{{justfile_directory()}}/$FLUTTER_WORKSPACE/android/src/main/jniLibs" build --release
+
+compile-android-local:
+    #!/usr/bin/env sh
+    set -eu
+    for TARGET in $ANDROID_TARGETS; do
+        {{just_executable()}} compile-android $TARGET
+    done
+
+flutter-run: dart-build
```

Comment:
Just moving a [previous discussion](https://github.com/xaynetwork/xayn_discovery_engine/pull/126#discussion_r803738386) into new PR
> @Robert-Steiner: In general we try to keep the PRs small and self-contained but you can also keep it in the pr.
But in this case at least flutter-build should also call compile-android-local if the target is apk. For flutter-run we could build both ios and android or also add a target there. wdyt?

Reply:
I'll double check how this currently works. I'm assuming it works in the CI because we build the libraries in the step before.